src/decisionengine/framework/tests/test_client_server.py

```diff
@@ -45,9 +45,15 @@ def test_client_print_product(deserver):
         "|  1 | value2 | 2      |\n" \
         "|  2 | value3 | Test   |\n" \
         "+----+--------+--------+"
+
+@pytest.mark.usefixtures("deserver")
+def test_client_print_product_not_real(deserver):
     output = deserver.de_client_run_cli('--print-product', 'NO_SUCH_PRODUCT')
-    assert output == "Product NO_SUCH_PRODUCT: Not produced by any module"
+    assert "Product NO_SUCH_PRODUCT:" in output
+    assert "Not produced by any module" in output
```

Comment:
It is not exactly the same. In theory, there could be both parts and not the whole and it would be a mistake. Should the original assert be after the 2 you added?

Reply:
I agree with @mambelli--unless the printout changed (i.e. inserting newline characters or whatever), the original test is better.

Reply:
Occasionally it kicks out text indicating the `test_channel` is offline as well.  I'll rework this with the original check at the end